### PR TITLE
fix: Updates how a description is added to a checkbox or radio

### PR DESCRIPTION
**Pull request by thiessenp-cds**
# Summary | Résumé

Originally for any checkbox or radio, a description was repeated for each one. This fix removes the repeated description. This works because the description is linked from the container fieldset to the description using an aria-describedby. The original code did not break WCAG but was bad UX for screen reader users.

Note:
**This doesn't solve the other problem of very verbose page/group loads**. We need to think of a strategy how to announce adding dynamic groups of stuff to a form. I suspect we could update the page title and use the live region announce component or similar. But then we'd want to remove the main live=polite on the main form element and that would require a lot of testing and thinking about edge cases e.g. show-hide rules.

## Test

Look at the source code for a checkbox or radio with and without a description added using the form-builder. If there is a description, it should only appear once (not in each input=radio/checkbox)

### CODE REVIEW

File: components/clientComponents/forms/MultipleChoiceGroup/MultipleChoiceGroup.tsx [hunk 1]
```diff
@@ -22,15 +22,15 @@ export const MultipleChoiceGroup = (props: MultipleChoiceGroupProps): React.Reac
         key={index}
         name={field.name}
         className={className}
-        ariaDescribedBy={ariaDescribedBy}
+        {...(ariaDescribedBy && { ariaDescribedBy: ariaDescribedBy })}
```
Comment (thiessenp-cds) [inline components/clientComponents/forms/MultipleChoiceGroup/MultipleChoiceGroup.tsx:25]: I can't think of a case where the Radio or Checkbox input's would need an aria-describedby. The description of the input I would think would be in it's related label and for anything lengthy and generic in the fieldset's legend and or description. But I guess if a specific checkbox really needed a lot of explaining then maybe it could be used. But I wonder if that checkbox would be packed with too much info/context.
So I left it in to be used optionally for future cases I can't think of.

Comment (timarney) [inline components/clientComponents/forms/MultipleChoiceGroup/MultipleChoiceGroup.tsx:25]: Should we just remove this ?

I don't see any code using the ariaDescribedBy for MultipleChoiceGroup

Comment (thiessenp-cds) [reply components/clientComponents/forms/MultipleChoiceGroup/MultipleChoiceGroup.tsx:25]: Sure. On second thought, leaving that there may cause more harm than good.

**Review by timarney — APPROVED** [review]
